scss/atoms/buttons/_mixins.scss

```diff
@@ -0,0 +1,64 @@
+/*
+* ==========================================================================
+* Buttons (Mixins)
+* ==========================================================================
+*/
+
+@mixin ui-button($color, $bgcolor, $disabled: false) {
+	background: $bgcolor;
+	border: 0;
+	color: $color;
+	display: inline-block;
+	font-weight: 500;
+	margin: .25em;
+	text-align: center;
+	text-decoration: none;
+	text-transform: uppercase;
+
+	@if($disabled) {
+		cursor: default;
```

Comment:
Oh wait, nevermind via email I thought you were commenting somewhere else, I shall adjust.